cs/src/Management/TunnelManagementClient.cs

```diff
@@ -91,7 +91,7 @@ public class TunnelManagementClient : ITunnelManagementClient
         public TunnelManagementClient(
             ProductInfoHeaderValue userAgent,
             Func<Task<AuthenticationHeaderValue?>>? userTokenCallback = null,
-            string? apiVersion = null)
+            string apiVersion = "2023-09-27-preview")
```

Comment:
Would this default version have to get updated every time we update the api version? Thoughts on making the value ` "2023-09-27-preview"` a constant variable/ put it in a centralized place so in the future we just need to update it in one place rather than a couple times in the same file

Reply:
good point, I added a constant